lib/Xud.ts

```diff
@@ -53,13 +53,28 @@ class Xud {
       await this.lndClient.connect();
 
       this.raidenClient = new RaidenClient(this.config.raiden);
+      await this.raidenClient.init();
 
       this.pool = new Pool(this.config.p2p, this.db);
-      this.pool.init();
 
       this.orderBook = new OrderBook(this.db.models, this.pool, this.lndClient);
       await this.orderBook.init();
 
+      const pairs: string[] = [];
+      (await this.orderBook.getPairs()).forEach((pair) => {
+        pairs.push(pair.id);
+      });
+
+      const handshakeData = {
+        pairs,
+        version: '1.0',
+        nodePubKey: this.nodeKey.nodePubKey,
+        listenPort: this.config.p2p.listen ? this.config.p2p.port : undefined,
+        raidenAddress: this.raidenClient.address,
+      };
+
+      this.pool.init(handshakeData);
```

Comment:
The `handshakeData` is used just once in this line. In my mind we don't need to define a new variable. So that the object gets created and the function called in one statement.